CompilerSource/backendflat/Background_generated.h

```diff
@@ -0,0 +1,295 @@
+// automatically generated by the FlatBuffers compiler, do not modify

```

Comment:
It amazes me that a Google project would write out such poor grammar... I sort of wish this was your doing....

Reply:
I don't understand? Where's the mistake? All I see is that it's technically not *automatic* because I had to invoke the compiler.

Reply:
Or do you mean the rest of this file?